v20.1/admin-ui-statements-page.md

```diff
@@ -7,135 +7,154 @@ toc: true
+**Rows Affected** displays statistics on rows returned for the SQL statement fingerprint.
 
-### Statistics by Gateway Node
+- **Mean Rows** is the average number of rows returned while executing statements with this fingerprint within the last hour or [specified time interval](#time-interval).
+- **Standard Deviation** is the value of one standard deviation of the mean.
 
-The **Statistics by Gateway Node** table provides a breakdown of the number of statements of the selected fingerprint per gateway node. For each gateway node, the table also provides the following details:
+### Diagnostics
 
-Parameter | Description
------|------------
-Node | The ID of the gateway node.
-Time | The cumulative time taken to execute the statement within the last hour or the [specified time interval](#limitation).
-Execution Count | The total number of times the SQL statement (or multiple statements having the same fingerprint) is executed.
-Retries | The cumulative number of retries to execute the SQL statement (or multiple statements having the same fingerprint) within the last hour or the [specified time interval](#limitation).
-Rows Affected | The average number of rows returned while executing the SQL statement (or multiple statements having the same fingerprint) within the last hour or the [specified time interval](#limitation). <br><br>The number of rows returned are represented in two ways: The numerical value shows the number of rows returned, while the horizontal bar is color-coded (blue indicates the mean value and yellow indicates one standard deviation of the mean value of the number of rows returned). The bar helps you compare the mean rows across all SQL fingerprints in the table. <br><br>You can sort the table by rows returned.
-Latency | The average service latency of the SQL statement (or multiple statements having the same fingerprint) within the last hour or the [specified time interval](#limitation). <br><br>The latency is represented in two ways: The numerical value shows the mean latency, while the horizontal bar is color-coded (blue indicates the mean value and yellow indicates one standard deviation of the mean value). The bar also helps you compare the mean latencies across all SQL fingerprints in the table. <br><br>You can sort the table by latency.
+The **Diagnostics** section of the Statement Details page allows you to activate and view diagnostics for the SQL statement fingerprint.
 
-### Execution Count
+When you activate diagnostics for a fingerprint, CockroachDB looks for SQL queries that match this fingerprint. On the next match, information about the SQL statement is written to a diagnostics bundle that you can download. This bundle consists of a JSON file that contains a distributed trace of the SQL statement. 
```

Comment:
is it worth clarifying here that the diagnostics will be collected a maximum of N times for a given activated fingerprint where N is the number of nodes in your cluster?